### ISSUE TITLE
`Focus Accessible Buffer` terminology and command ID are still present

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#195403]
The terminal now uses the accessible view, so the concept of `Accessible Buffer` should be removed, but I don't want to break existing command IDs / custom key bindings 🤔.

https://github.com/microsoft/vscode/blob/33876c27f082973adbddc027e0e532c3648099dd/src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibilityHelp.ts#L78-L87

cc @Tyriar

**Comment by meganrogge** [comment]
I just realized this was needed to allow for the `ctrl/cmd+UpArrow` navigation. https://github.com/microsoft/vscode/blob/33876c27f082973adbddc027e0e532c3648099dd/src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminal.accessibility.contribution.ts#L221
